tests/test_ingest.py

```diff
@@ -81,3 +81,16 @@ def test_run(self):
         ingest.run()
         config.config('agent')['backup_mode'] = True
         ingest.run()
+
+    def test_get_config_params(self):
+        properties = '\n'.join([
+            'org.opencastproject.workflow.config.encode_720p=true',
+            'org.opencastproject.workflow.config.cutting=false',
+            'org.opencastproject.workflow.definition=fast',
+            'org.opencastproject.nonsense=whatever'
+            ])
+        workflow, parameters = ingest.get_config_params(properties)
+        self.assertEqual(workflow, 'fast')
+        self.assertIn(('encode_720p', 'true'), parameters)
+        self.assertIn(('cutting', 'false'), parameters)
+        self.assertEqual(len(parameters), 2)
```

Comment:
This is a little bit nicer:

```suggestion
        self.assertEqual([('encode_720p', 'true'), ('cutting', 'false')],
                         parameters)
```

Reply:
I think the following result would be valid:

```py
[('cutting', 'false'), ('encode_720p', 'true')]
```

But that's something your test would fail on. We could use sets if you prefer that:

```py
self.assertEqual(set([('encode_720p', 'true'), ('cutting', 'false')]), set(parameters))
```

Reply:
Yes, you are correct. If the order does not matter, sets would be better.

I personally find the later with the sets more readable than splitting it up in 3 tests, but I am okay with both.

Choose one and feel free to merge then

Reply:
Okay, switched to using sets